### PR TITLE
Add multiline tag support

### PR DESCRIPTION
**Pull request by grenbeg**
Inspired by https://github.com/rtts/djhtml/issues/115

- Respects multiline template tags as valid tags
- Indents new lines inside multiline template tag, except closing tag lines

Tests pass successfully
image

### CODE REVIEW

**Comment by grenbeg** [comment]
When applied to Qromponents repo, creates this diff:
image

File: tests/suite/django.html [hunk 1]
```diff
@@ -195,3 +195,32 @@ <h1>Welcome, {{ request.user }}</h1>
 {% customtag %}
     <div></div>
 {% endcustomtag %}
+
+<!-- Two-line custom tag -->
```
Comment (dave-v) [inline tests/suite/django.html:199]: Are these all valid formats?

**Review by dave-v — APPROVED** [review]

File: tests/suite/django.html (hunk 1 above)
Comment (grenbeg) [reply tests/suite/django.html:199]: Do you mean valid according to this PR, or valid according to Qromponents?

Comment (grenbeg) [reply tests/suite/django.html:199]: the answer is yes either way 😄

Comment (dave-v) [reply tests/suite/django.html:199]: I meant, does this file contain only "valid" formats, according to djhtml?

Comment (grenbeg) [reply tests/suite/django.html:199]: yeah, running the test suite unindents every line in that file, applies djhtml, and asserts equal with the original file. tests succeed with the examples added to that html file

**Pull request merged**